src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataEvaluator.cs

```diff
@@ -205,17 +207,5 @@ private static ViewDataInfo GetPropertyValue(object container, string propertyNa
 
             return new ViewDataInfo(container, propertyInfo, () => propertyInfo.GetValue(container));
```

Comment:
FYI the lambda here (really the compiler-generated class instances) amounts to a bit over 4.75 mbyte. Calling `GetValue()` up front would save all of that but would change the semantics significantly. Leaving it alone for now.

Sometime we should probably have a high-level look at our value accessor / value duality in `ViewDataInfo` and `ModelExplorer`.